### PR TITLE
Cve 2020 1728

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1728.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "keycloak is vulnerable to a None"
+                "value": "It was found that pages on the Admin Console area of the application are completely missing general HTTP security headers in HTTP-responses. This does not directly lead to a security issue, yet it might aid attackers in their efforts to exploit other problems. The flaws unnecessarily make the servers more prone to Clickjacking, channel downgrade attacks and other similar client-based attack vectors."
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1728.json]: Based on the description it is unclear which software is affected.

Please mention it.

File: 2020/1xxx/CVE-2020-1728.json [hunk 2]
```diff
@@ -4,15 +4,68 @@
...
+                {
+                    "vendor_name": "[UNKNOWN]",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "keycloak",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1728.json]: CVE entries must contain version information ([8.1.1](https://cve.mitre.org/cve/cna/rules.html#section_8-1_cve_entry_information_requirements))

File: 2020/1xxx/CVE-2020-1728.json [hunk 3]
```diff
@@ -4,15 +4,68 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "It was found that pages on the Admin Console area of the application are completely missing general HTTP security headers in HTTP-responses. This does not directly lead to a security issue, yet it might aid attackers in their efforts to exploit other problems. The flaws unnecessarily make the servers more prone to Clickjacking, channel downgrade attacks and other similar client-based attack vectors."
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1728.json]: CVE entry descriptions must contain enough information for a reader to identify the affected product(s) ([8.2.1](https://cve.mitre.org/cve/cna/rules.html#section_8-2_cve_entry_prose_description_requirements))

File: 2020/1xxx/CVE-2020-1728.json (hunk 2 above)
Comment (cristina479) [reply 2020/1xxx/CVE-2020-1728.json]: If all versions of Keycloak are affected, the version_value field should indicate "All versions" or something similar. Please make the proper changes.

**Pull request merged**